docs/vars.md

```diff
@@ -64,6 +64,7 @@ following default cluster paramters:
 * *dns_setup* - Enables dnsmasq
 * *dnsmasq_dns_server* - Cluster IP for dnsmasq (default is 10.233.0.2)
 * *skydns_server* - Cluster IP for KubeDNS (default is 10.233.0.3)
+* *skydns_server_secondary* - Secondary Cluster IP for KubeDNS (default is 10.233.0.4)
```

Comment:
Concerning cluster upgrades, would there be any guarantee that something else isn't already using 10.233.0.4 in a cluster? Seems like any service IP from `10.233.0.0/18` could be potentially taken if it wasn't reserved in the original cluster deployment.

Reply:
I looked over the kubespray code, and apparently there is no special way that ip adresses are locked/guaranteed.
According to the code, the dns ip address is always made using this: `skydns_server: "{{ kube_service_addresses|ipaddr('net')|ipaddr(3)|ipaddr('address') }}"` which is by default put into both `k8s-cluster.yml` and the defaults file. So I did the same with the secondary ip address.
If the 4th ip requested as here is then added to cluster-dns to the kubelet, the ip will not be served/used for services.

If you do not use CoreDNS the 4th ip is not locked / used for anything and thereby useable inside the cluster.

I hereby say that there should be no problem at all with this method of use in kubespray deployments.